docs/builtin_elements.md

```diff
@@ -578,6 +578,18 @@ FIXME: write docs
 
 FIXME: write docs
 
+The `Flickable` is a lower-level item that is the base for scrollable
+elements, such as the ScrollView widget. When the viewport-width or the
+viewport-height is greater than the parent width or parent height
+respectively the element becomes scrollable although the `Flickable`
+does not create a scrollbar. When using a for loop to populate the
```

Comment:
But one should say that it does automatically computes it based on its contents. With the exception of the bug if it is a for loop